app/controllers/editions_controller.rb

```diff
@@ -343,7 +348,12 @@ def redirect_url
   end
 
   def make_govuk_url_relative(url = "")
-    url.sub(%r{^https?://(www\.)?gov\.uk/}, "/")
+    url.sub(%r{^(https?://)?(www\.)?gov\.uk/}, "/")
+  end
+
+  def validate_redirect(redirect_url)
+    regex = /\A\/([a-z0-9]+-)*[a-z0-9]+\z/
```

Comment:
This only allows a `/` at the start, which would prevent redirecting to (eg) `/topic/keeping-farmed-animals`

Reply:
Good point! Thanks. Should be ok now 🤞